packages/api/src/app-router.ts

```diff
@@ -260,7 +262,8 @@ export default async function makeApp(params: CliArgs) {
   if (fallbackProxy) {
     app.use(proxy({ target: fallbackProxy, changeOrigin: true }));
   }
-  app.use(errorHandler());
+  const wwwHandler = await require("@livepeer.studio/www");
```

Comment:
Dos this import need to be dynamic? Could have it as 
`import wwwPromise from '...'`
on top then here just 
`.use(await wwwPromise)`

It's generally better for TypeScript inference (and maybe some build tooling) so I generally try to avoid raw `require` function.